Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Handler.yml

```
@@ -241,20 +241,216 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "7":
+    continueonerrortype: ""
+    form:
+      description: Add one or more tags to the assets(s) associated with the current Cortex XSOAR incident.
+      expired: false
+      questions:
+      - defaultrows: []
+        fieldassociated: ""
+        gridcolumns: []
+        id: "0"
+        label: ""
+        labelarg:
+          simple: What tag(s) do you want to add to asset(s)?
+        options: []
+        optionsarg: []
+        placeholder: ""
+        readonly: false
+        required: false
+        tooltip: Specify a comma-separated list of tags you want to add to the asset(s) associated to the current Cortex XSOAR incident.
+        type: shortText
+      sender: Your SOC team
+      title: Determine tags to add to asset(s)
+      totalanswers: 0
+    id: "7"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    message:
+      bcc:
+      body:
+      cc:
+      format: ""
+      methods: []
+      subject:
+      timings:
+        completeafterreplies: 1
+        completeaftersla: false
+        completeafterv2: true
+        retriescount: 0
+        retriesinterval: 0
+      to:
+    nexttasks:
+      '#none#':
+      - "9"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 54995d00-c6c0-4549-8ded-319d5bc01e0e
+      iscommand: false
+      name: Determine tags to add to asset(s)
+      type: collection
+      version: -1
+      description: Add one or more tags to the assets(s) associated with the current Cortex XSOAR incident.
+    taskid: 54995d00-c6c0-4549-8ded-319d5bc01e0e
+    timertriggers: []
+    type: collection
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1020
+        }
+      }
+  "9":
+    continueonerrortype: ""
+    id: "9"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "2"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      asm_id_list:
+        complex:
+          accessor: xpanseassetids
+          root: incident
+          transformers:
+          - args:
+              chars:
+                value:
+                  simple: '[]'
+            operator: StripChars
+          - args:
+              limit: {}
+              replaceWith: {}
+              toReplace:
+                value:
+                  simple: '"'
+            operator: replace
+      tags:
+        complex:
+          accessor: Answers}
+          root: ${Determine tags to add to asset
```

In a complex format, no need for `${<key>}` format
```suggestion
          root: Determine tags to add to asset
```

---

changed in 
[sasha recommendations](https://github.com/demisto/content/pull/35111/commits/1d4310c1fe899d397e962890e60dd244edd74b83)